core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -1037,24 +1104,35 @@ class ZkMigrationIntegrationTest {
     admin.alterUserScramCredentials(alterations)
   }
 
-  def verifyTopicConfigs(zkClient: KafkaZkClient): Unit = {
-    TestUtils.retry(10000) {
+  def verifyTopicConfigs(zkClient: KafkaZkClient, shouldRetry: Boolean): Unit = {
+    maybeRetry(shouldRetry, 10000) {
       val propsAfter = zkClient.getEntityConfigs(ConfigType.TOPIC, "test")
       assertEquals("204800", propsAfter.getProperty(TopicConfig.SEGMENT_BYTES_CONFIG))
       assertFalse(propsAfter.containsKey(TopicConfig.SEGMENT_MS_CONFIG))
     }
   }
 
-  def verifyClientQuotas(zkClient: KafkaZkClient): Unit = {
-    TestUtils.retry(10000) {
-      assertEquals("1000", zkClient.getEntityConfigs(ConfigType.USER, Sanitizer.sanitize("user@1")).getProperty("consumer_byte_rate"))
-      assertEquals("900", zkClient.getEntityConfigs(ConfigType.USER, "<default>").getProperty("consumer_byte_rate"))
-      assertEquals("800", zkClient.getEntityConfigs("users/" + Sanitizer.sanitize("user@1") + "/clients", "clientA").getProperty("consumer_byte_rate"))
-      assertEquals("100", zkClient.getEntityConfigs("users/" + Sanitizer.sanitize("user@1") + "/clients", "clientA").getProperty("producer_byte_rate"))
-      assertEquals("10", zkClient.getEntityConfigs(ConfigType.IP, "8.8.8.8").getProperty("connection_creation_rate"))
+  def verifyBrokerConfigs(zkClient: KafkaZkClient, shouldRetry: Boolean): Unit = {
+    maybeRetry(shouldRetry, 10000) {
+      val defaultBrokerProps = zkClient.getEntityConfigs(ConfigType.BROKER, "<default>")
+      assertEquals("86400000", defaultBrokerProps.getProperty(ServerLogConfigs.LOG_RETENTION_TIME_MILLIS_CONFIG))
+
+      val broker0Props = zkClient.getEntityConfigs(ConfigType.BROKER, "0")
+      assertEquals("43200000", broker0Props.getProperty(ServerLogConfigs.LOG_RETENTION_TIME_MILLIS_CONFIG))
+
+      val broker1Props = zkClient.getEntityConfigs(ConfigType.BROKER, "1")
+      assertEquals("43200000", broker1Props.getProperty(ServerLogConfigs.LOG_RETENTION_TIME_MILLIS_CONFIG))
     }
   }
 
+  def verifyClientQuotas(zkClient: KafkaZkClient): Unit = {
```

Comment:
why removing the retry? It seems that make the `testDualWriteQuotaAndScram` unstable

Reply:
Ah good catch! I did not mean to change this logic. Will fix.